packages/components/src/button/index.tsx

```diff
@@ -65,10 +65,9 @@ function useDeprecatedProps( {
 	}
 
 	if ( isDefault ) {
-		deprecated( 'Button isDefault prop', {
+		deprecated( 'wp.components.Button `isDefault` prop', {
```

Comment:
Any idea why we don't have `deprecated()` calls for the rest of the variant old props (like `isSmall`, `isPrimary`, etc.)?

Reply:
I can't find an explicit reason being given, but it does look [intentional](https://github.com/WordPress/gutenberg/pull/31713/commits/c03043105b11db6dce7a285aafcff965f4ef5091).

Reply:
Looks like we just wanted to soft-deprecate first, and to hard-deprecate later: https://github.com/WordPress/gutenberg/pull/31713#issuecomment-845489235

It's been almost 3 years, so maybe it's about time for hard deprecation.